### PR TITLE
Add API.post service.

### PR DESCRIPTION
**Pull request by maxpeterson**
For https://incuna-pm.atlassian.net/browse/OP-88

### CODE REVIEW

**Comment by maxpeterson** [comment]
@incuna/frontend please review / merge

File: web/app/scripts/src/services/api.es.js [hunk 1]
```diff
@@ -33,6 +33,12 @@ module.service(moduleProperties.componentName, [
             getForm: function (url) {
                 return $http.get(url)
                     .then((response) => response.data);
+            },
+            post: function (baseUrl, data) {
+                const endpoint = 'respond';
```
Comment (henrahmagix) [inline web/app/scripts/src/services/api.es.js]: Can you allow setting of `endpoint` in the configuration as a provider?

Comment (wytrych) [inline web/app/scripts/src/services/api.es.js]: I don't think it's a better solution than this, it's still out of sync with backend. A better solution would be to pass in the endpoints from the backend somehow.

Comment (maxpeterson) [inline web/app/scripts/src/services/api.es.js]: We have `const endpoint = 'forms';` just above. Seems a bit pointless to do this differently.

Comment (henrahmagix) [inline web/app/scripts/src/services/api.es.js]: They should all be able to be set via a provider =)

Comment (wytrych) [inline web/app/scripts/src/services/api.es.js]: I don't think it's such a great solution. Manually setting endpoints is not great, either way. The real solution would be to get the list of endpoints from the backend. But we can fix that later.

**Comment by maxpeterson** [comment]
Included in #34

**Pull request closed without merging**